mod/ckeditor/views.php

```diff
@@ -2,7 +2,6 @@
 
 return [
 	'default' => [
-		'ckeditor.js' => __DIR__ . '/vendors/ckeditor/ckeditor.js',
```

Comment:
FWIW, I still see value in renaming `elgg/ckeditor.js` because its presence forces us to use CKEDITOR_BASEPATH.